kubernetes/structure_persistent_volume_spec.go

```diff
@@ -373,9 +373,9 @@ func flattenPersistentVolumeSpec(in v1.PersistentVolumeSpec) []interface{} {
 	if in.PersistentVolumeReclaimPolicy != "" {
 		att["persistent_volume_reclaim_policy"] = in.PersistentVolumeReclaimPolicy
 	}
-	if in.StorageClassName != "" {
-		att["storage_class_name"] = in.StorageClassName
-	}
+
+	att["storage_class_name"] = in.StorageClassName
```

Comment:
This should not be a breaking change requiring resource re-creation as `storage_class_name` is marked `Optional` and `Computed`:

https://github.com/terraform-providers/terraform-provider-kubernetes/blob/v1.10.0/kubernetes/schema_persistent_volume_claim.go#L81-L87